server/api/balancer.go

```diff
@@ -61,6 +63,75 @@ func (h *balancerHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	h.rd.JSON(w, http.StatusOK, balancersInfo)
 }
 
+type operator struct {
```

Comment:
Do we need to support other operators here? if not, I suggest using peerOperator.